tests/react-test-app/android/app/build.gradle

```diff
@@ -88,7 +88,7 @@ android {
 
     defaultConfig {
         applicationId "io.realm.react.testapp"
-        minSdkVersion 16
+        minSdkVersion 19
```

Comment:
I believe that React Native 0.60.0 has `minSdkVersion` set to 16. If we go with 19 (Android 4.4), we need to document it somewhere.

Reply:
this is our test app. why do you think it needs document somewhere? 
The minSDKVersion change here does not mean we are deprecating Android versions

Reply:
For the example, it is fine.